repipe

```diff
@@ -41,6 +41,20 @@ usage() {
     echo "validate          Validatei pipeline instead of set pipeline"
     echo "validate-strict   Validate pipeline with strict mode"
     echo "non-interactive   Run set-pipeline in non-interactive mode"
+    echo "url               Pipeline dashboard URL. Usage: ci/repipe url | xargs -L1 open"
+}
+
+show_pipeline_url() {
+	spruce merge --skip-eval pipeline.yml ${settings_file} > .deploy.yml
```

Comment:
Using `--skip-eval` so that the command can be fast, and not slowed down by a dozen lookups to Vault.